pom.xml

```diff
@@ -110,15 +110,66 @@
             <version>${postgresql.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <!-- All packages added by ojdbc8-production. Do not use ojdbc8-production directly to avoid .pom file in final package -->
```

Comment:
nit : - please add all packages imported by ojdbc8-production explicitly to pom when bumping up driver version?